ports/stm32/boards/MIKROE_QUAIL/mpconfigboard.h

```diff
@@ -0,0 +1,113 @@
+#define MICROPY_HW_BOARD_NAME       "MikroE Quail"
+#define MICROPY_HW_MCU_NAME         "STM32F427VI"
+#define MICROPY_PY_SYS_PLATFORM     "MikroE Quail"
+
+// 1 = use STM32 internal flash (1 MByte)
+// 0 = use onboard external SPI flash (8 MByte)
+#define MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE (0)
+
+#define MICROPY_HW_ENABLE_RNG       (1)
+#define MICROPY_HW_ENABLE_RTC       (1)
+#define MICROPY_HW_ENABLE_USB       (1)
+#define MICROPY_HW_HAS_FLASH        (1)
+
+// HSE is 12MHz
+#define MICROPY_HW_CLK_PLLM         (6)
+#define MICROPY_HW_CLK_PLLN         (336)
+#define MICROPY_HW_CLK_PLLP         (RCC_PLLP_DIV4)
+#define MICROPY_HW_CLK_PLLQ         (14)
+#define MICROPY_HW_CLK_LAST_FREQ    (1)
+
+// The board has no crystal for the RTC
+#define MICROPY_HW_RTC_USE_LSE      (0)
+#define MICROPY_HW_RTC_USE_US       (0)
+#define MICROPY_HW_RTC_USE_CALOUT   (0)  // turn on/off PC13 512Hz output
+
+// UART config
+// mikroBUS slot 1
+#define MICROPY_HW_UART3_NAME       "SLOT1"
```

Comment:
Are these good names, do they match something written on the board itself?

Reply:
Well, as you can see from the [picture](https://github.com/micropython/micropython-media/blob/143855a2f0852f87fe9b5037d6a61488747888e3/boards/MIKROE_QUAIL/quail_top.jpg), each slot is named "mikroBUS" plus a big number 1 to 4. `SLOT1` is concise and pretty obvious, I believe. I was inspired by another MikroE board supported by MicroPython, [CLICKER2](https://github.com/micropython/micropython/tree/master/ports/stm32/boards/MIKROE_CLICKER2_STM32).

I don't mind change the name. What would you suggest? `MIKROBUS1`, `MIKRO1`, `BUS1`, `MB1`?

Reply:
Ah, the same names are used by the CLICKER2 board definition.  So, yes, keep them as they are.

Reply:
OK.